about.html

```diff
@@ -0,0 +1,197 @@
+</head>
+
+<body>
+	<header >
+		<nav class="header-nav">
+			<div class="container-top-nav">
+				<div class="header-top-nav">
+					<ul class="header-top-nav-desktop">
+						<li><a href="#" class="fa fa-facebook"></a></li>
+						<li><a href="#" class="fa fa-twitter"></a></li>
+						<li><a href="#">English</a></li>
+						<li><a href="#">My Page</a></li>
+						<li><a href="#">Logout</a></li>
+					</ul>
+				</div>
+			</div>
+			<div class="container-nav">
+				<div class="logo-nav">
+					<a href="./index.html">
+						<img src="./img/logo/ethio-health-logo.png" alt="logo Ethio health">
+					</a>
+				</div>
+				<ul class="header-nav-desktop">
+					<li><a href="./about.html">About</a></li>
+					<li><a href="#">Program</a></li>
+					<li><a href="#">Join</a></li>
+					<li><a href="#">Sponsor</a></li>
+					<li><a href="#">News</a></li>
+					<li><a href="#">Ticket</a></li>
+				</ul>
+			</div>
+		</nav>
+		<div class="container">
+			<div class="header-nav-phone">
+				<button type="button" class="menu-btn">
+					<img src="./img/icon/menu-icon.svg" alt="menu icon">
+				</button>
+				<nav class="header-nav-mobile hidden">
+					<button type="button" class="btn-close-nav">
+						<img src="./img/icon/close-icon.svg" alt="cancel icon">
+					</button>
+					<ul>
+						<li>
+							<a href="./index.html" >
+								Home
+							</a>
+						</li>
+						<li>
+							<a href="./about.html" >
+								About
+							</a>
+						</li>
+						<li>
+							<a href="#" >
+								Program
+							</a>
+						</li>
+						<li>
+							<a href="#" >
+								Join
+							</a>
+						</li>
+						<li>
+							<a href="#" >
+								Sponsor
+							</a>
+						</li>
+						<li>
+							<a href="#" >
+								News
+							</a>
+						</li>
+						<li>
+							<a href="#" >
+								Ticket
+							</a>
+						</li>
+					</ul>
+				</nav>
+			</div>
+			<div class="headline">
+				<h3>
+					"Hello! Sharing world"
+				</h3>
+				<h1>
+					ETHIO HEALTH <br>EXHIBITION AND CONGRESS 2023
+				</h1>
+				<p class="headline-desc">
+					Ethio Health is the premier, largest and comprehensive healthcare, medical, pharmaceutical and wellness technology,
+					inputs and solutions international trade fair in Ethiopia.
+				</p>
+				<div class="headline-desc-desktop">
+					<p>Ethio Health is the premier, largest and comprehensive healthcare, medical, pharmaceutical and wellness technology,
+					   inputs and solutions international trade fair in Ethiopia.
+					</p>
+					<p>
+						This event has proved to be the major platform that fuels the sector development through trade facilitation, market
+						linkages, introduction of new technology and transfer of knowledge for concerned stakeholders across the whole value
+						chain.
+					</p>
+				</div>
+				<div class="headline-contact">
+					<p>
+						Please contact us per Email for any further questions about Ethio Health Exhibition 2023!
+					<br>
+						<a>marketing@pranevents.net</a>
+					</p>
```

Comment:
- [ ] Pleased make sure that the email here is underlined like in the design
![image](https://user-images.githubusercontent.com/74744787/218277487-26847f64-9f86-4e28-882b-c966312579da.png)
.